packages/workbox-precaching/controllers/PrecacheController.mjs

```diff
@@ -219,12 +219,14 @@ class PrecacheController {
     const tempCache = await caches.open(this._getTempCacheName());
 
     const requests = await tempCache.keys();
-    await Promise.all(requests.map(async (request) => {
+    // Process each request/response one at a time, deleting the temporary entry
+    // when done, to help avoid triggering quota errors.
+    for (const request of requests) {
```

Comment:
+1 to this change. I think it'll make it a lot easier to respond to errors in a predictable way.